src/pages/Login/Login.module.scss

```diff
@@ -0,0 +1,153 @@
+@mixin formLabel{
+    font-family: Biryani;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 1rem;
+    line-height: 160%;
+    letter-spacing: 0.1em;
+    color: #FFFFFF;
+}
+
+@mixin formInputField {
+    outline: none;
+    background: #FFFFFF;
+    border: 2px solid #E6E6E6;
+    box-sizing: border-box;
+    border-radius: 4px;
+    font-weight: 600;
+    font-size: 2rem;
+}
+
+@mixin formGroup {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
+
+.overlay{
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: #595757;
+
+    & .loginOverlay{
```

Comment:
You seem to missunderstood me regarding this symbol &
When it comes to scss.
```
<div className= "overlay">
     <div className = "overlayLoginOverlay">
     </div>
</div>
```

and writing css 
```
.overlay{
     &LoginOverlay{

     }
}
```

this sign & = overlay so when writing &LoginOverlay{} you actually writing overlayLoginOverlay{}. [info](https://sass-lang.com/documentation/style-rules/parent-selector)

So writing like this `& .loginOverlay` selector does nothing. The reason we use this selector is for easier css maintainability. So that the developer could see if it's nested or not.

So easy fix would be is to:
1) Rename classes like this from .loginOverlay to overlayLoginOverlay.
2) Delete this & selector. If you don't want to add long css classes.

Reply:
@ArnasDickus 

Sorry I did misunderstood what you mean. However, I do have a question. In the example you gave, what if there is a deep nesting in jsx? wouldn't name be too long for `className`? For example I have 3 levels deep nesting with `div` then each child `div` className is added on top of it's parent's className.

I like your suggestion but also concern about the length of naming.

Reply:
@tomneo2004 my issue is that you write scss like this `& .loginOverlay` you add a space between & and .loginOverlay. Which makes this & selector useless. I think you right about naming concerns. Using this & is more useful when writing [BEM scss](http://getbem.com/introduction/). However, since module.scss is already enclosed we don't need to do that. So just remove this & selector from your code, since as I mentioned it doesn't do anything with added space. [documentation](https://sass-lang.com/documentation/style-rules/parent-selector)

Reply:
@ArnasDickus Ok. I will remove `&` from scss file and leave everything unchanged.

Reply:
Yes, I agree that makes sense